canonical-websites/www.ubuntu.com
# Create openstack features page

## Done
Created the openstack/features page and added it to the navigation.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/features](http://0.0.0.0:8001/openstack/features)
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#)

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/3289
